### PR TITLE
Fix CI tests failure.

### PR DESCRIPTION
**Pull request by bozhiyou**

### CODE REVIEW

File: lonestar/analytics/cpu/bipart/Coarsening.cpp [hunk 1]
```diff
@@ -29,7 +29,7 @@
 #include <unordered_set>
 #include <unordered_map>
 
-constexpr static const unsigned CHUNK_SIZE      = 512U;
+// constexpr static const unsigned CHUNK_SIZE = 512U;
```
Comment (nicelhc13) [inline lonestar/analytics/cpu/bipart/Coarsening.cpp:32]: Let's remove this.

File: lonestar/analytics/cpu/bipart/Coarsening.cpp [hunk 2]
```diff
@@ -310,8 +311,9 @@ void moreCoarse(MetisGraph* graph, galois::LargeArray<unsigned>& weight) {
               fineGGraph->getData(e).setMatched();
               fineGGraph->getData(e).setParent(nn);
               fineGGraph->getData(e).netnum = fineGGraph->getData(b).netnum;
-           		//fineGGraph->getData(e).netnum = fineGGraph->getData(b).netnum.load();
-						 }
+              // fineGGraph->getData(e).netnum =
```
Comment (nicelhc13) [inline lonestar/analytics/cpu/bipart/Coarsening.cpp:314]: Let's remove this.

File: lonestar/analytics/cpu/bipart/Coarsening.cpp [hunk 3]
```diff
@@ -237,24 +237,25 @@ void parallelHMatchAndCreateNodes(MetisGraph* graph, int iter, GNodeBag& bag,
...
           bag.push(nodeid);
           unsigned ww = 0;
           for (auto pp : edges) {
             ww += fineGGraph->getData(pp).getWeight();
             fineGGraph->getData(pp).setMatched();
             fineGGraph->getData(pp).setParent(nodeid);
             fineGGraph->getData(pp).netnum = fineGGraph->getData(item).netnum;
-         		//fineGGraph->getData(pp).netnum = fineGGraph->getData(item).netnum.load();
-				  }
+            // fineGGraph->getData(pp).netnum =
```
Comment (nicelhc13) [inline lonestar/analytics/cpu/bipart/Coarsening.cpp:249]: Let's remove this.

File: lonestar/analytics/cpu/bipart/Coarsening.cpp [hunk 4]
```diff
@@ -44,40 +44,40 @@ int hash(unsigned val) {
...
       galois::steal(),
-	//		 galois::chunk_size<CHUNK_SIZE>());
+      // galois::chunk_size<CHUNK_SIZE>());
       galois::loopname("rand_index"));
-		T_INDEX.stop();
+  T_INDEX.stop();
 
-		//std::cout <<"rand: " << T_RAND.get() << std::endl;
-		//std::cout << "rand_index: " << T_INDEX.get() << std::endl;
+  // std::cout <<"rand: " << T_RAND.get() << std::endl;
```
Comment (nicelhc13) [inline lonestar/analytics/cpu/bipart/Coarsening.cpp:79]: Let's remove this.

**Comment by nicelhc13** [comment]
I will remove unnecessary comments from Coarsening.

**Comment by bozhiyou** [comment]
> I will remove unnecessary comments from Coarsening.

There are more "unnecessary comments" in the `bipart` app. I only fixed the memory leak issue and let the authors remove them.

**Pull request merged**